config.sample.yaml

```diff
@@ -24,11 +24,34 @@ transform:  # Optional
     - url: https://github.com/MY_TRANSFORM_CODE_REPO.git
     - url: https://github.com/MY_OTHER_TRANSFORM_CODE_REPO.git
       sha: abcd01abcd01abcd01abcd01abcd01abcd01abcd
-  datalake:
+  datalake:  # Optional
     zones:
       - Bronze
       - Silver
       - Gold
+  spark_config:  # Optional
+    - key: spark.databricks.cluster.profile  # Configuration for a Single Node cluster
+      value: singleNode
+    - key: spark.master
+      value: local[*]
+  databricks_libraries:  # Optional
+    pypi:
+      - package: opencensus-ext-azure==1.1.9
+      - package: opencensus-ext-logging==0.1.1
+        repo: "custom-mirror"
+    maven:
+      - coordinates: "com.amazon.deequ:deequ:1.0.4"
+        repo: "custom-mirror"
+        exclusions: ["org.apache.avro:avro"]
+    cran:
+      - package: "rkeops"
+        repo: "my-awesome-repo"
+    whl:
+      - "dbfs:/FileStore/baz.whl"
+    egg:
+      - "dbfs:/FileStore/foo.egg"
+    jar:
+      - "dbfs:/FileStore/app-0.0.1.jar"
```

Comment:
Wanted to check if these would work currently or if they are stubs? From what I can tell only the pypi is being tested as working in a deployment

Reply:
These all should work, yes. I have tested that the configuration for all of them is propagated to the cluster

We currently only use pypi though, but I felt that it would be better to add all of the supported libraries now, in case they are needed in the future

Reply:
seems reasonable overall, only concern I would have is that if we had a user who's expecting these to be used out of the box. I'd be tempted to add a comment so its clear the config values aren't used as of yet or just keep the pypi and maven as those are the ones we have specific plans for